src/java.base/share/classes/sun/security/util/KeyUtil.java

```diff
@@ -174,27 +174,6 @@ public static final int getKeySize(AlgorithmParameters parameters) {
         return -1;
     }
 
-    /**
-     * Returns the algorithm name of the given key object. If an EC key is
-     * specified, returns the algorithm name and its named curve.
-     *
-     * @param key the key object, cannot be null
-     * @return the algorithm name of the given key object, or return in the
-     *       form of "EC (named curve)" if the given key object is an EC key
-     */
-    public static final String fullDisplayAlgName(Key key) {
-        String result = key.getAlgorithm();
-        if (key instanceof AsymmetricKey ak) {
-            AlgorithmParameterSpec paramSpec = ak.getParams();
-            if (paramSpec instanceof NamedCurve nc) {
-                result += " (" + nc.getNameAndAliases()[0] + ")";
-            } else if (paramSpec instanceof NamedParameterSpec nps) {
-                result = nps.getName();
-            }
-        }
-        return result;
-    }
-
```

Comment:
Would you consider to keep this common method and modify it to return the key size and algorithm name?